src/components/Tracker/elements/Encounters/Encounters.tsx

```diff
@@ -96,11 +99,31 @@ const Encounters = React.memo(function Encounters() {
   const handleScroll = (index: number) => {
     listRef.current?.scrollToItem(index + 1, 'center');
   };
+  
+  const navigate = useNavigate();
+
+  const handleOpen = (e: React.MouseEvent<HTMLDivElement, MouseEvent>, index: number) => {
+    e.preventDefault();
+    e.stopPropagation();
+    navigate(`/badgedetail/${selectedGame?.value}/${index}`);
```

Comment:
What happens here if value doesn't exist?

Reply:
It'll just redirect back to the previous page.